README.md

```diff
@@ -56,7 +56,7 @@ In the Magento root directory
 
 - Install the module
 ```
-composer require vpietri/adm-quickdevbar
+composer require vpietri/adm-quickdevbarc --dev
```

Comment:
hi, there is a typo accidentally appended with an extra "c"

Reply:
Whoops, sorry about that. Fixed!